README.md

```diff
@@ -88,13 +92,13 @@ None of the types below need to be constructed directly; they arise from perform
 
     Type for representing the identity map of a certain size `M=N`, obtained simply as `IdentityMap{T}(M)`, `IdentityMap(T,M)=IdentityMap(T,M,N)=IdentityMap(T,(M,N))` or even `IdentityMap(M)=IdentityMap(M,N)=IdentityMap((M,N))`. If `T` is not specified, `Bool` is assumed, since operations between `Bool` and any other `Number` will always be converted to the type of the other `Number`. If `M!=N`, an error is returned. An `IdentityMap` of the correct size and element type will automatically be created if `LinearMap` objects are combined with `I`, Julia's built in identity (`UniformScaling`).
 
-*   `LinearCombination`, `CompositeMap`, `TransposeMap` and `CTransposeMap`
+*   `LinearCombination`, `CompositeMap`, `TransposeMap` and `AdjointMap`
 
     Used to add and multiply `LinearMap` objects, don't need to be constructed explicitly.
 
 ## Examples
 
-The `LinearMap` object combines well with the iterative eigensolver `eigs`, which is the Julia wrapper for Arpack.
+The `LinearMap` object combines well with the iterative eigensolver `eigs`, which is the Julia wrapper for Arpack, and with iterative solvers from [IterativeSolvers.jl](https://github.com/JuliaMath/IterativeSolvers.jl).
 
```

Comment:
This should now probably mention to `[Arpack](https://github.com/JuliaLinearAlgebra/Arpack.jl)`